data-raw/ischemic_stroke.R

```diff
@@ -0,0 +1,27 @@
+library(tidymodels)
+library(janitor)
+library(forcats)
+
+# ------------------------------------------------------------------------------
+
+tidymodels_prefer()
+theme_set(theme_bw())
+options(pillar.advice = FALSE, pillar.min_title_chars = Inf)
+
+# ------------------------------------------------------------------------------
+
+load(url("https://github.com/topepo/FES/raw/master/Data_Sets/Ischemic_Stroke/stroke_data.RData"))
```

Comment:
```suggestion
load(url("https://github.com/topepo/FES/raw/06812c48a21882808403cee338b8312fdbd35a46/Data_Sets/Ischemic_Stroke/stroke_data.RData"))
```

Let's use a permalink here!